UPGRADING.md

```diff
@@ -1,6 +1,15 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 0.18.1
+
+#### Aligned and clarified callback behavior of 405 handler and built-in OPTIONS handler
+
+When a request is made to a resource using an undefined method (HTTP 405) or a request is
+made to the built-in `OPTIONS` handler, only the `before` callbacks associated with the
+resource will be run.  Other callbacks (`before_validation`, `after_validation`, `after`)
```

Comment:
You're the boss.  :)  I added another commit that restores `after` but still skips the _validation callbacks as well as parameter processing for the built-in `OPTIONS` handler.